test/unit/spec/EnrollChoices_spec.js

```diff
@@ -567,14 +567,14 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
         return setup(resAllFactors).then(function (test) {
           var factorList = test.form.getFactorList();
           expect(factorList).toEqual(['OKTA_VERIFY', 'U2F', 'WINDOWS_HELLO', 'YUBIKEY', 'GOOGLE_AUTH',
-            'SMS', 'CALL', 'QUESTION', 'DUO', 'SYMANTEC_VIP', 'RSA_SECURID', 'GENERIC_SAML', 'GENERIC_OIDC']);
```

Comment:
Question for UI folks: What actually determines the factorList order? It doesn't seem to match the order in MFA_ENROLL_allFactors.js.

Does it even matter?

Reply:
It matters. We have a sort order defined in [FactorUtil](https://github.com/okta/okta-signin-widget/blob/master/src/util/FactorUtil.js#L24)

Reply:
Ah, I see. Side note, does it make sense to you to change the order so that password is first, given its importance? Or is it alright to leave it in the order that it is now?

Reply:
This is intentional, password is not first since it is not one of the strongest factors. We've had a few discussions and iterations of this with PMs. Take a look at our docs for authentication methods https://help.okta.com/en/prod/Content/Topics/Security/MFA.htm